Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm

```diff
@@ -302,7 +302,7 @@ void requestLinkDecorationFilteringData(LinkFilteringRulesCallback&& callback)
 {
     m_cachedQuirks = WTFMove(quirks);
     m_cachedQuirks.shrinkToFit();
-    RELEASE_LOG_ERROR(ResourceLoadStatistics, "StorageAccessPromptQuirkController::setCachedQuirks: Loaded %lu storage access prompt(s) quirks from WebPrivacy.", quirks.size());
+    RELEASE_LOG_ERROR(ResourceLoadStatistics, "StorageAccessPromptQuirkController::setCachedQuirks: Loaded %lu storage access prompt(s) quirks from WebPrivacy.", m_cachedQuirks.size());
```

Comment:
Oh, while you're here — maybe we can downgrade this to a regular `RELEASE_LOG`? (It's unclear why this would be an error)